### ISSUE TITLE
Divulgação/apresentação de campeonatos

### ISSUE THREAD

**Issue by mrcaique** [CalicoUFSC/ufsc-cco-reformulado#10]
Uma ideia que tive seria a apresentação/divulgação das competições relacionadas à computação: programação ([ACM International Collegiate Programming Contest](https://icpc.baylor.edu/), [CodeChef](https://www.codechef.com/), etc.), Machine Learning ([Kaggle](https://www.kaggle.com/), [HackerRank](https://www.hackerrank.com/domains/ai/machine-learning), etc.), etc..

Outro ponto que vale é citar (se possível) os conhecimentos necessários para as competições, como por exemplo, álgebra linear é um conhecimento necessário para machine learning, portanto, seria útil mostrar se a universidade oferece tais cursos e como cursá-los (seja no próprio curriculo como na universidade em si, como os cursos de verão e disciplinas extracurriculares).

Uma dúvida que tenho é como que seria realizada essa divulgação, quais os meios?\* Qual o público destinado? A esta pergunta, penso principalmente nos calouros, no entanto, vale para todos.

A motivação que me veio por isso é porque vejo que os exercícios são a melhor forma de praticar os conhecimentos e, como as competições trazem exercícios usados nas respectivas áreas, o aluno teria ideia de como a teoria é aplicada e, talvez, ganhe o interesse na área. Inclusive, seria um complemento às disciplinas que não oferecem o conteúdo totalmente ou o conteúdo seja "abstrato".

Geralmente essas competições têm ligações com Universidades (no caso da ACM também com a sociedade científica educacional), seria bom mostrar em quais a Ufsc trabalha. Senão (e se for possível) realizar competições internas ou treinamentos.

Claro que isto tudo depende do interesse do aluno, então a divulgação boa é uma forma de interessá-lo.*

Por ora é isto que sugiro. O que vocês têm a dizer?

<sub>Obs.: 
1) Sim, eu sei que em Fundamentos da Matemática Discreta e Teoria da Computação citam a maratona de programação, inclusive, até penso que a disciplina Introdução à Computação também pode ser um bom local de divulgação.
2) Este é um empecilho que encontrei, não sei como seria "divulgar bem". Só salientando que eu penso nas competições mais pela fonte de conhecimento que ela pode oferecer.
3) Sinceramente, tive dúvidas se esta issue foge do escopo principal do repositório que é reformulação do currículo.</sub>

**Comment by ranisalt** [comment]
Até gosto da ideia de incentivar participação nesses campeonatos, mas temos que deixar claro que isso deve ser completamente opcional.

A prof. Jerusa, que orienta os grupos que vão participar, talvez consiga abrir uma bolsa apenas para dedicar os alunos aos treinamentos, o que já ajuda bastante, já que hoje os alunos que participam das maratonas fazem isso no tempo livre, geralmente curto.

Não foge do escopo não, é uma atividade que tem bastante a ver com o curso e é algo a se por em pauta também :+1:

**Comment by mrcaique** [comment]
Sim, a ideia é mostrar que existe muito mais além do que a universidade oferece, como eu disse, tudo depende do interesse do aluno (seja por participar, seja por aprender).

Pelo que já conversei com a professora Jerusa, se conseguir, ela disse que pretendia usar os desafios da maratona de programação como exercícios na disciplina optativa de Tópicos Especiais em Algoritmos (não sei se é I ou II), assim sendo, já acaba sendo um bom treino e/ou mais conhecimento.
